### ISSUE TITLE
Img Sizing 

### ISSUE THREAD

**Issue by j-p-courneya** [Relay-Maryland/relaymaryland.com#24]
@brianzelip 

Is there a way to govern the size of images so styling controls their size?

**Comment by brianzelip** [comment]
@j-p-courneya Yes there is, and it's currently set up to handle most use cases you'd throw at it.

I'm not sure what you have in mind. If you're referring to the Town Hall pic, that pic starts out breaking the rules by being very small, both size and resolution. It works though decent enough. After you added the image, I added centering of all images in the pages that you might add in case they didn't take up the entire column width.
